students.py

```diff
@@ -0,0 +1,63 @@
+import os
+import pickle
+ 
+class Students(object):
+    
+    def __init__(self,name):
+        
+        self.filename = name
+        self.key = ('name','surname','age')
+        
+        if self.filename in os.listdir():
+            with open(self.filename, 'rb') as f:
+                self.group_students = self.fileinfo()
```

Comment:
Зачем тут читать файл, если вы его, по сути, не читаете тут, а  читаете в методе `fileinfo`.
```suggestion
            self.group_students = self.fileinfo()
```